README.md

```diff
@@ -1,2 +1,15 @@
 # ExpenseTrackerGrupo5
 Take control of your finances.
+
+
+### Levantar contenedores 
```

Comment:
all the documentation has to be in English

Reply:
Sure